app/policies/partner_policy.rb

```diff
@@ -54,12 +54,13 @@ def permitted_attributes
              { calendars_attributes: %i[id name source strategy place_id partner_id _destroy],
                address_attributes: %i[id street_address street_address2 street_address3 city postcode],
                service_areas_attributes: %i[id neighbourhood_id _destroy],
-               tag_ids: [] }]
+               tag_ids: [], category_ids: [], facility_ids: []  }]
```

Comment:
Do we still need `tag_ids: []`? that may present a bypass if somehow something submitted a partnership ID as a basic tag, without the user being a partnership admin?